google/datalab/contrib/pipeline/_pipeline.py

```diff
@@ -167,13 +176,16 @@ def _get_operator_classname(task_detail_type):
       with the string "Operator".
     """
     task_type_to_operator_prefix_mapping = {
+      'bash': 'Bash',
```

Comment:
Can we just target the minimal scope and only implement what is needed for that? Will make it easier to review and get the basics right first.

So we should have just the BQ operators needed to implement %bq pipeline.

Reply:
Bash is actually useful for some quick local testing, so I'll keep that (there is no implementation; this will just pass-through to airflow the way the current code is designed). 

Similarly, the other airflow operators help in testing when I need to eliminate issues with the new operators that have been implemented. 

I've removed a few aliases now though.